Resources/about.json

```diff
@@ -0,0 +1,20 @@
+{
+	"appInfo": {
+		"appName": "SwiftIsland",
+		"appVersion": "1.1.1",
+		"appDescription": "Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet.",
+		"organisers": [
+			"@funky-monkey",
```

Comment:
Might be good to make this an object so we have customization possitiblities. E.g.
```
{
  "handle": "",
  "name": "",
  "twitter": "",
  "url": "",
  "linkedin": ""
}
```
What do you think?

Reply:
👍